src/ert/config/ensemble_config.py

```diff
@@ -119,31 +117,62 @@ def get_gen_kw_templates(config_dict: ConfigDict) -> list[tuple[str, str]]:
     @no_type_check
     @classmethod
     def from_dict(cls, config_dict: ConfigDict) -> EnsembleConfig:
-        grid_file_path = config_dict.get(ConfigKeys.GRID)
+        # Grid file handling:
+        # Each field can specify its own grid file, or fall back to a global grid.
+        # If neither is provided, validation will fail when processing fields.
+        global_grid_file_path = config_dict.get(ConfigKeys.GRID)
```

Comment:
I guess it's not part of this PR, but when having two `GRID ....` files in my config I haven't got any errors.

Reply:
> I just tried the same on `main`, namely running the heat-equation with two `GRID` keywords in the config, and I get no warnings. Is this what you did as well or are you talking about something else? If so, I think this can be a new issue.

Yes, that's what I meant. Based on the docs it is not really clear how it should be used: https://ert.readthedocs.io/en/latest/reference/configuration/keywords.html#id8
I guess a new issue for the warning.

Additionally, the current documentation should be updated too: https://ert.readthedocs.io/en/latest/reference/configuration/keywords.html#field

Reply:
I have updated the docs.